plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-fields-block/style.scss

```diff
@@ -62,6 +62,7 @@
 }
 
 .wc-block-components-address-form__address_2-toggle {
+	display: inline-block;
```

Comment:
This fixes a display bug on the address form `+ Add apartment, suite, etc.` toggle where it is not spaced correctly on mobile screens.

Reply:
That button is not switched to be a span, how did this fix it?